src/main/java/dk/cngroup/lentils/controller/ClientContactController.java

```diff
@@ -18,7 +18,8 @@
 @Controller
 @RequestMapping("/contact")
 public class ClientContactController {
-    private static final String VIEW_CONTACT = "client/contact";
+    private static final String CLIENT_VIEW_CONTACT = "client/contact";
```

Comment:
Všude je je prefix podobných konstant jen VIEW_ a tady máš CLIENT_VIEW. Má to nějaký důvod?

Reply:
nemá :). Vše změním na VIEW_CLIENT_* :+1: